docs/.vuepress/config.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
                 '/sigs/Build-System',
                 '/sigs/Cloud',
                 '/sigs/Core',
+				'/sigs/HPCandAI',
```

Comment:
```suggestion
                '/sigs/HPCandAI',
```

Makes it consistent with spaces